snapcraft/extensions/gnome.py

```diff
@@ -24,8 +24,8 @@
 
 from .extension import Extension, get_extensions_data_dir, prepend_to_env
 
-_SDK_SNAP = {"core22": "gnome-42-2204-sdk"}
-_PLATFORM_TRANSLATION = {"core22": "2204"}
+_SDK_SNAP = {"core22": "gnome-42-2204-sdk", "core24": "gnome-46-2404"}
```

Comment:
how come this is `"gnome-46-2404"` and not `"gnome-46-2404-sdk"?

Reply:
good catch!